config/kubernetes-sigs/sig-scheduling/teams.yaml

```diff
@@ -37,6 +37,18 @@ teams:
     members:
     - k82cn
     privacy: closed
+  kube-scheduler-simulator-admins:
+    description: Admin access to the kube-scheduler-simulator
+    members:
+    - adtac
+    - alculquicondor
+    - Huang-Wei
+    privacy: closed
+  kube-scheduler-simulator-maintainers:
+    description: Write access to the kube-scheduler-simulator
+    members:
+    - sanposhiho
```

Comment:
Add you also add @adtac here?

Reply:
Sure. How about you and @alculquicondor ?

Reply:
We are already owners, so I don't really see the need for write access.

Reply:
I don't need to be included.

This option is for "enforced" writing/merging, so is used only there is a particular need.

Reply:
> This option is for "enforced" writing/merging, so is used only there is a particular need.

I see. added @adtac 👌